### ISSUE TITLE
[FR] Option so Julia start's in the same directory as the file I opened

### ISSUE THREAD

**Issue by azev77** [JunoLab/Juno.jl#458]
It would be awesome if there was an option I could set, so in the future whenever I open a file w/ Juno I will automatically be in the directory of that file w/o needing `cd()`. 

Those who don't want this option as default, don't need to select it.

**Comment by pfitzseb** [comment]
So do you really want to start Julia in the current file's directory or have Julia basically follow the current file's directory (so `cd` before evaluating)?

**Comment by azev77** [comment]
@pfitzseb yes, it would be very useful to have the default option to start Julia in the current file's directory. 
That would make it similar to RStudio/Matlab/Jupyter etc. 
Of course I still have the flexibility to change directories after starting Julia. 

Maybe, there could also be an option in Juno (though I wouldn't find it useful): 
1 click here so **as a default**, Juno starts Julia in the current file's directory
2 click here so **as a default**, Juno starts Julia in "desired directory"

**Comment by pfitzseb** [comment]
So currently Julia starts in the top-most project's directory and then stays in that directory unless the user changes it. You're proposing to instead `cd` to the directory the currently focused file is in, but also to never change? That doesn't seem super useful to me unless you restart your Julia session really often.

**Comment by azev77** [comment]
Sorry if I wasn't clear.
1a Right now if I open a Julia program w/ Juno and type `pwd()` I get 
> C:\\Users\\azevelev

1b If I open a Julia notebook in same folder w/ Jupyter and type `pwd()` I get
> C:\\Users\\azevelev\\Dropbox\\COMPUTATION\\_QE_Moll\\MLEcon_SovereignDefault\\benchmarkingML-master\\Sovereign_Default\\Julia

2 I would like it so that whenever I open a Julia program in Juno I'm automatically in the same directory as that folder. Many other IDEs have this. It would be particularly helpful to newcomers to Julia using Juno.
